lib/ansible/module_utils/basic.py

```diff
@@ -272,6 +272,11 @@
     sys.exit(1)
 
 
+_NO_MODIFY_KEYS = frozenset(
+    ('msg', 'exception', 'warnings', 'deprecations', 'failed', 'skipped', 'changed', 'results')
```

Comment:
rc, stdout, stderr .. 

'results' should not happen on module side, that is appended on loop on controller

Reply:
```suggestion
    ('msg', 'exception', 'warnings', 'deprecations', 'failed', 'skipped', 'changed', 'rc', 'stdout', 'stderr')
```